src/PageLayout/Notification/Recurring/RecurringDeliveryRequest2.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import truck from '../../../icons/green_truck.svg';
+
+class RecurringDeliveryRequest2 extends Component {
+    render() {
+        return (
+            <div className="modal-wrapper">
+                
+                <img className="icon" src={truck} alt="icon" />
+                <div className="modal-left-align">
+                    <h1>Are you sure you want to claim this pickup?</h1>
+                    <h2 className="warning">Once a pickup is claimed it cannot be cancelled.</h2>
+                    <p><span className="to-from">From:</span> Local Point      <span className="right to-from">To:</span> Seattle Union Gospel Mission <br/>
+                        {/* TODO: Backend replace info with timestamp */}
+                    Start Date: Thursday, 11/21/2018
+                    10 Pickups requested for every Thursday
+                    Pickup between 10 am -12 pm</p>
+                    <div className="popup-buttons">
+                        <button onClick={this.props.nextStep} className="claim" type="button">Confirm</button> <button onClick={this.props.close} className="reject" type="button">Cancel</button>
```

Comment:
this Confirm button is where we'll actually write to DB to really claim the delivery for this user. So:
1. Add a placeholder function `claimDelivery()` in `RecurringDeliveryRequestController` with a TODO where we actually write to db
2. The `onClick` here should call a local function in this file that first calls `this.props.claimDelivery` and then call `this.props.nextStep`